diff --git a/packages/games/tools/retrorun/package.mk b/packages/games/tools/retrorun/package.mk
index 94a0350c..e4f6934a 100644
--- a/packages/games/tools/retrorun/package.mk
+++ b/packages/games/tools/retrorun/package.mk
@@ -7,4 +7,4 @@ PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/navy1978/retrorun"
 PKG_URL="${PKG_SITE}.git"
-PKG_DEPENDS_TARGET="toolchain libdrm libpng linux libevdev librga openal-soft"
+PKG_DEPENDS_TARGET="toolchain libdrm libpng linux libevdev librga openal-soft curl ${OPENGLES}"
 PKG_TOOLCHAIN="make"
@@ -16,5 +16,7 @@ pre_configure_target() {
     local INC_FLAGS="-I$(get_build_dir libdrm)/include/drm -I$(get_build_dir linux)/include/uapi -I$(get_build_dir linux)/tools/include"
   fi
+  INC_FLAGS+=" -I$(get_build_dir ${OPENGLES})/include"
+  INC_FLAGS+=" -I$(get_build_dir ${OPENGLES})/include/GBM"
   CFLAGS+=" ${INC_FLAGS}"
   CXXFLAGS+=" ${INC_FLAGS} -std=gnu++17"
 }
